my-app/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <v-data-table :headers="headers" :items="orders" hide-actions class="elevation-1">
+      <template slot="items" slot-scope="props">
```

Comment:
where do you have the `items` `<slot>` used? 
@andiwolfpack  @robikovacs  correct me if I'm wrong, but I think you need this code somewhere:
```
<items>
   <slot></slot>
</items>
```
and then your items from here will be inserted there. If you don't have this, is there a reason to use slots here?

Reply:
@florinionce that is the name of the slot, `"items"`, the code is correct. Check "Scope slots" in [here](https://vuetifyjs.com/en/components/data-tables#api)

Reply:
Oh, so the slot is defined by vuetify. Thanks :D

Reply:
Correct @florinionce !